kDriveCore/Data/Models/File.swift

```diff
@@ -730,13 +731,20 @@ public final class File: Object, Codable {
         return ProxyFile(driveId: driveId, id: id)
     }
 
+    public static func uid(driveId: Int, fileId: Int) -> String {
```

Comment:
To truly be a UID, shouldn't we use the user ID also ?

Reply:
As you guessed, `driveId` is unique so no need to go further with `userId`.